integ-test/src/test/java/org/opensearch/sql/sql/QueryIT.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.sql;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_BEER;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+public class QueryIT extends SQLIntegTestCase {
+    @Override
+    public void init() throws IOException {
+        loadIndex(Index.BEER);
+    }
+
+    @Test
+    public void all_fields_test() throws IOException {
+        String query = "SELECT * FROM "
+                + TEST_INDEX_BEER + " WHERE query('*:taste')";
+        JSONObject result = executeJdbcRequest(query);
+        assertEquals(16, result.getInt("total"));
+    }
+
+    @Test
+    public void mandatory_params_test() throws IOException {
```

Comment:
Would be nice to have some more tests, like `optional_params_test`...

Reply:
Fixed in fe67f0168906e58e0b61e1f3f6f9072f14c19b67